app/src/ui/multi-commit-operation/choose-branch/base-choose-branch-dialog.tsx

```diff
@@ -273,7 +276,7 @@ export abstract class BaseChooseBranchDialog extends React.Component<
           <DropdownSelectButton
             selectedValue={operation}
             options={this.getMergeOptions()}
-            disabled={!this.canStart}
```

Comment:
It'd be nice if our linter or tsc warned us about falsey/truthy checks 🤔

Reply:
We could enable this: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md